ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -1015,19 +1003,22 @@ private class JdbcLedgerDao(
           Ok
       }
 
+    val txId = Some(ledgerEntry.entry) collect {
+      case x: LedgerEntry.Transaction => x.transactionId
+    }
```

Comment:
```suggestion
    val txId = ledgerEntry.entry match {
      case x: LedgerEntry.Transaction => Some(x.transactionId)
      case _ => None
    }
```
I realized afterwards that the meaning is the same, but maybe the alternative is a bit more straightforward for the reader. Feel free to discard.

Reply:
I definitely find the suggested code easier to read.

Reply:
that's actually an artefact from me trying to debug something. `txId` is not used anywhere. thanks for bringing it to my attention. I'll delete it.